tools/requirements.txt

```diff
@@ -0,0 +1,2 @@
+ffmpeg-python==0.1.17
```

Comment:
I think you will still need to add something to the Travis configuration to install these requirements. Currently it still runs `pip install flake8 jsonschema`, so I don't think it will install ffmpeg. I also think flake8 should be added here, if you decide to make that change.

Reply:
I understand I have to add a line `pip install flake8 jsonschema ffmpeg-python`, is that right? I think this library depends on the ffmpeg binaries and if they are not available this module just will not work.

Reply:
True, you will have to install the ffmpeg binaries package in the config file as well: https://docs.travis-ci.com/user/installing-dependencies/